src/utilities.ts

```diff
@@ -86,6 +86,10 @@ export async function setCachedToolPath(toolName:string, version : string) {
 
 export function getDownloadUrl(toolName : string, version: string): string {
     const system = os.type()
+    if (system == "Linux") {
+        const arch = os.arch()
+        system = system+"_"+arch
```

Comment:
since system is a const this assignment will not work. Also could you space out the string concatenation like system + "_" + arch.

A good alternative to this could be

```TypeScript
const system = os.type();
const systemAndArch = system == "Linux" ? `${system}_${os.arch()}` : system;
```

Reply:
Thanks for those feedback.
I made the changes locally but I am not sure how to run the test locally before pushing them to be sure that it is correct, can you guide me ?
Regarding the CLA, of course. I am just formally securing it with my employer before really signing (even if I got an oral validation)

Reply:
Sure! Simply run `npm install` to install all node_modules (if you haven't already) then `npm test`. This is assuming you have Node installed locally.

Reply:
I think the test are correct now. Thanks to your help I have been able to run npm test correctly.